modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableViewSkinBase.java

```diff
@@ -816,18 +820,24 @@ private void refreshView() {
         }
     }
 
-    // Handles the horizontal scrolling when the selection mode is cell-based
-    // and the newly selected cell belongs to a column which is not totally
-    // visible.
+    /**
+     * Handles the horizontal scrolling when the selection mode is cell-based
+     * and the newly selected cell belongs to a column which is not totally
+     * visible.
+     */
     void scrollHorizontally() {
```

Comment:
This method should be `protected`, and so should be `onFocusPreviousCell`, `onFocusNextCell`, `onSelectPreviousCell`, and `onSelectNextCell`.